src/mesh/Mesh.cpp

```diff
@@ -416,6 +416,42 @@ void Mesh:: computeState()
   computeBoundingBox();
 }
 
+void Mesh:: buildBoundingBox()
```

Comment:
What does this method do compared to `computeBoundingBox()`. Why are both needed? Why are the normals computed here?

Reply:
We needed a method to compute bounding boxes only and can be called from solverInterface. If I remember correctly, some time a go (maybe version 1.3) we had only one big method computeState, but now this method calls two other methods: computeBoundBox and computeNormals. At that time I extracted the parts we needed and wrapped them into buildBoundingBox.

In addition, it seems that we do not need the normals here. 

But now, computeBoundingBox() does the samething, the only problem is it is a private method and can't be called from solverInterface. We can either make it public or include the mesh.hpp in solverInterface. right?

Reply:
We should avoid the code duplication. Let's make `computeBoundingBox()` public then. I will anyway iterate over `computeState()` again, I guess.